storybook/pages/InviteFriendsToCommunityPopupPage.qml

```diff
@@ -1,96 +1,137 @@
+                Utils.globalUtilsInst = this
+                globalUtilsReady = true
+
+            }
+            Component.onDestruction: {
+                globalUtilsReady = false
+                Utils.globalUtilsInst = {}
+            }
         }
-        Component.onDestruction: {
-            mainModuleReady = false
-            Utils.mainModuleInst = {}
+
+        QtObject {
+            function getContactDetailsAsJson() {
+                return JSON.stringify({})
+            }
+
+            Component.onCompleted: {
+                mainModuleReady = true
+                Utils.mainModuleInst = this
+            }
+            Component.onDestruction: {
+                mainModuleReady = false
+                Utils.mainModuleInst = {}
+            }
         }
-    }
 
-    Loader {
-        active: globalUtilsReady && mainModuleReady
-        anchors.fill: parent
+        Loader {
+            id: loader
+            active: globalUtilsReady && mainModuleReady
+            anchors.fill: parent
 
-        sourceComponent: InviteFriendsToCommunityPopup {
-            parent: parent
-            modal: false
-            anchors.centerIn: parent
+            sourceComponent: InviteFriendsToCommunityPopup {
+                parent: parent
+                modal: false
+                anchors.centerIn: parent
 
-            community: ({
-                id: "communityId",
-                name: "community-name"
-            })
+                community: ({
+                    id: "communityId",
+                    name: "community-name"
+                })
 
-            rootStore: QtObject {
-                function communityHasMember(communityId, pubKey) {
-                    return false
+                rootStore: QtObject {
+                    function communityHasMember(communityId, pubKey) {
+                        return false
+                    }
                 }
-            }
 
-            contactsStore: QtObject {
-                readonly property ListModel myContactsModel: ListModel {
-                    Component.onCompleted: {
-                        for (let i = 0; i < 20; i++) {
-                            const key = `pub_key_${i}`
-
-                            append({
-                                alias: "",
-                                colorId: "1",
-                                displayName: `contact ${i}`,
-                                ensName: "",
-                                icon: "",
-                                isContact: true,
-                                localNickname: "",
-                                onlineStatus: 1,
-                                pubKey: key
-                            })
+                communitySectionModule: QtObject {
+                    function inviteUsersToCommunity() {
+                        logs.logEvent("communitySectionModule::inviteUsersToCommunity",
+                                      ["keys", "message"], arguments)
```

Comment:
How does this work? Shouldn't the function have these 2 arguments? (keys, message)

Reply:
Yeah I know that but shouldn't the function have this signature: 
```JS
function inviteUsersToCommunity(keys, message)
```

Reply:
I added `keys` and `message` as you wrote to avoid confusion. But technically speaking it's the same, js doesn't care if parameters are named or not.